test/benchmark/discr/discr_mappings_test.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2014, 2015 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+# External
+import pytest
+import numpy as np
+
+# Internal
+import odl
+from odl.discr.grid import sparse_meshgrid
+from odl.discr.discr_mappings import (
+    GridCollocation, NearestInterpolation, LinearInterpolation,
+    PerAxisInterpolation)
+
+
+pytestmark = pytest.mark.benchmark
+
+
+def initialize():
+    global shape, values, hcube, grid, space, dspace32, dspace64
```

Comment:
Alternative would be to just not make it a function. But I thought it might be useful to have a figure on the setup time, too. Or maybe the function could return all that stuff.